docs/src/demos/hooks/use-viewport-size.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { useViewportSize } from '@mantine/hooks';
+import { Text, Group } from '@mantine/core';
+
+const code = `
+import { useViewportSize } from '@mantine/hooks';
+import { Text, Group } from '@mantine/core';
+
+function Demo() {
+  const { height, width } = useViewportSize();
+
+  return (
+    <Group position="center">
+      <Text>
+        Viewport width: {width}
+      </Text>
+      <Text>
+        Viewport height: {height}
+      </Text>
+    </Group>
+  );
+}
+`;
+
+function Demo() {
+  const { height, width } = useViewportSize();
+
+  return (
+    <Group position="center">
```

Comment:
There is no need in Text and Group components, just a simple div will do the job

```tsx
<div>Width: {width}, height: {height}</div>
```